.github/workflows/ci.yml

```diff
@@ -137,6 +136,10 @@ jobs:
           invoke deps --gx-install -m '${{ matrix.markers }}' -r test
 
       - name: Run the tests
+        env:
```

Comment:
Maybe I should only set these env variables for the cloud mark stage?

Reply:
There may be a way to conditionally inject env vars with gh actions but another simpler solution could be to just break the steps that need different env vars to their own job.

Reply:
Yeah, that sounds like a good idea.

Reply:
I'm going to leave it like this for this PR and move it out later if we want.

Reply:
Actually, I will move this out now. Setting the environmental variables cause a lot of issues. I will file a followup ticket to review this further. It only impacts people who are both gx cloud and OSS developers.